Knossos.NET/ViewModels/GlobalSettingsViewModel.cs

```diff
@@ -800,7 +806,11 @@ internal void SaveCommand()
             {
                 blMirrors.Add("aigaion.feralhosting.com");
             }
-            if(blMirrors.Any() && blMirrors.Count() != 3 /*Invalid!*/)
+            if (BlTalos)
+            {
+                blMirrors.Add("talos.feralhosting.com");
+            }
+            if (blMirrors.Any() && blMirrors.Count() != 3 /*Invalid!*/)
```

Comment:
Should this be 4 now? And I think we have to add Talos to the else statment below.

Reply:
yeah you right